src/SmartComponents/FormComponents/DescriptiveButton.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
+import './descriptiveCheckbox.scss';
```

Comment:
There is still a bunch of code copied from `DescriptiveCheckbox` which is not needed in this case (styles imports, and all checkbox-related stuff). Please, try to clean it a bit.